src/main/java/me/zeroeightsix/kami/mixin/client/MixinRenderManager.java

```diff
@@ -18,14 +19,20 @@
  */
 @Mixin(value = RenderManager.class, priority = 114514)
 public class MixinRenderManager {
-
+    Minecraft mc = Minecraft.getMinecraft();
     @Shadow
     public boolean renderOutlines;
     @Shadow
     public boolean debugBoundingBox;
 
     @Inject(method = "renderEntity", at = @At("HEAD"), cancellable = true)
     public void renderEntityPre(Entity entity, double x, double y, double z, float yaw, float partialTicks, boolean debug, CallbackInfo ci) {
+        if (!entity.equals(mc.player) && NoRender.INSTANCE.isEnabled() &&
+                (NoRender.INSTANCE.getEntityList().contains(entity.getClass()) ||
+                        NoRender.INSTANCE.getEntityList().contains(entity.getClass().getSuperclass()))) {
```

Comment:
What are the consequences of checking the super class here?

Reply:
this was useful for checking if an entity is a mob. 
I tried to add the direct superclasses of every mob to the EntityList when the mob option was enabled.
This was dirty and i am currently changing that. 
I cant handle the mobs with the hashset anymore since i have to instanceof like checks which is not possible with contains() so i will handle them separately. 

i will treat the mobs in their own if  Statements and remove the superclass check